okta-hosted-login/README.md

```diff
@@ -0,0 +1,66 @@
+# Express.js & Okta-Hosted Login Page Example
+
+This example shows you how to use the [oidc-middleware library] to login a user.  The login is achieved through the [OAuth 2.0 authorization code flow], where the user is redirected to the Okta-Hosted login page.  After the user authenticates, they are redirected back to the application.
+
+
+## Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Org, you can sign up for one at https://developer.okta.com/signup/.
+* An OIDC application in your Org, configured for Web mode. You can find instructions [here][OIDC Web Application Setup Instructions].  When following the wizard, use the default properties.  They are are designed to work with our sample applications.
+
+[OIDC Web Application Setup Instructions]: https://developer.okta.com/authentication-guide/implementing-authentication/auth-code#1-setting-up-your-application
+
+[OAuth 2.0 authorization code flow]: https://developer.okta.com/authentication-guide/implementing-authentication/auth-code
+
+## Running This Example
+
+To run this application, you first need to clone this repo and then enter into this directory:
+
+```bash
+git clone git@github.com:okta/samples-nodejs-express-4.git
+cd samples-nodejs-express-4/okta-hosted-login
+```
+
+Then install dependencies:
+
+```bash
+npm install
+```
+
+You will need to provide these values to the sample application:
+
+* Client ID
+* Client Secret
+* Issuer
+
+These settings can be found in the Developer Console, when looking at the Application that you created.  Place them into the file `.samples.config.json` in this folder:
```

Comment:
Might be a bigger conversation, but we could probably restructure the keys in `samples.config.json`.